`kernel-specification.yml` missing from source distribution

### Version of Awkward Array

2.10.0

### Description and code to reproduce

It seems that the `kernel-specification.yml` needed by the test suite is not included in the source distribution:

```pytb
============================================================== FAILURES ===============================================================
____________________________________________ test_cuda_compute_wrappers_match_kernel_spec _____________________________________________

    def test_cuda_compute_wrappers_match_kernel_spec():
>       spec = _spec_signatures()
               ^^^^^^^^^^^^^^^^^^


tests/test_4056_cuda_compute_signature_consistency.py:68: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

    def _spec_signatures():
        """kernel/specialization name -> list of spec argument names."""
>       with open(SPEC) as f:
             ^^^^^^^^^^
E       FileNotFoundError: [Errno 2] No such file or directory: '/tmp/portage/dev-python/awkward-2.10.0/work/awkward-2.10.0/kernel-spec
ification.yml'


tests/test_4056_cuda_compute_signature_consistency.py:28: FileNotFoundError
````

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -89,6 +89,7 @@ include = [
     "/CONTRIBUTING.md",
     "/README.md",
     "/juliapkg.json",
+    "/kernel-specification.yml",
     "/requirements-test-full.txt",
     "/requirements-test-minimal.txt"
 ]

